contentwarehouse/snippets/get_document_schema_test.py

```diff
@@ -0,0 +1,38 @@
+# # Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from contentwarehouse.snippets import get_document_schema_sample
+from contentwarehouse.snippets import test_utilities
+
+import pytest
+
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "us"
+document_schema_id = "78jim5u8cvrrg"
```

Comment:
What if this isn't there (because another test deleted it)?

Suggest merging all the current tests into one, so they can create a schema, work with it, and finally delete it, testing each step.

Reply:
I agree with combining the tests. By returning values from the samples, it'll make it easier to handle the create/delete operations